components/availability/src/Availability.svelte

```diff
@@ -115,6 +113,14 @@
           }
         }
 
+        if (
+          availability === AvailabilityStatus.PARTIAL &&
+          freeCalendars.length <
+            allCalendars.length * (participant_threshold / 100)
+        ) {
+          availability = AvailabilityStatus.BUSY;
+        }
```

Comment:
Change `partial` to `busy` if the # of participants for the timeSlot falls below the set `participant_threshold`